src/components/job-overview/index.js

```diff
@@ -172,25 +174,27 @@ const JobOverview = (props) => {
                             envName: deployment.environment,
                           })}
                         >
-                          {deployment.environment}
+                          <Typography link as="span">
+                            {deployment.environment}
+                          </Typography>
                         </Link>
-                      </p>
+                      </Typography>
                     ))}
                   {job.components && (
                     <ComponentList components={job.components}></ComponentList>
                   )}
                 </div>
-              </div>
-              <div className="job-overview__stepslist">
+              </section>
+              <section>
```

Comment:
add grid spacing for child elements
```suggestion
              <section className="grid grid--gap-medium">
```

Reply:
fixed